UNRELEASED.md

```diff
@@ -14,7 +14,4 @@
 
 ### Code quality
 
-- Removed various type assertions and bumped test coverage ([#2638](https://github.com/Shopify/polaris-react/pull/2638))
```

Comment:
Did you miss this one? Or did you cherry pick it out?

Reply:
It was mistakenly added back in https://github.com/Shopify/polaris-react/pull/2800

Reply:
https://github.com/Shopify/polaris-react/pull/2819/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR49

Reply:
Here it is: https://github.com/Shopify/polaris-react/blob/master/CHANGELOG.md#code-quality